src/Microsoft.AspNetCore.Authentication.OpenIdConnect/OpenIdConnectHandler.cs

```diff
@@ -335,21 +423,21 @@ protected virtual Task<bool> HandleSignOutCallbackAsync()
 
             GenerateCorrelationId(properties);
 
-            var redirectContext = new RedirectContext(Context, Options, properties)
+            var redirectContext = new RedirectContext(Context, Scheme, Options, properties)
             {
                 ProtocolMessage = message
             };
 
-            await Options.Events.RedirectToIdentityProvider(redirectContext);
+            await Events.RedirectToIdentityProvider(redirectContext);
             if (redirectContext.HandledResponse)
             {
                 Logger.RedirectToIdentityProviderHandledResponse();
-                return true;
+                return;
             }
             else if (redirectContext.Skipped)
             {
                 Logger.RedirectToIdentityProviderSkipped();
-                return false;
+                return;
```

Comment:
We'll have to revisit what skip means, this change makes it the same as Handled.